walk-in-interview/src/main/webapp/strings.en.js

```diff
@@ -37,3 +43,37 @@ AppStrings['homepage'] = {
   'filter-by-submit': 'Apply Filters',
   'job-listings-title': 'Job Listings:',
 };
+
+/**
+ * All the strings to be used in the new-job folder.
+ */
+AppStrings['new-job'] = {
```

Comment:
any reason why you want all the strings to be in one place? shouldn't new-job strings be inside the new-job package?

Reply:
later it in update-job feature, many strings are the same, so simply merge into one